integration_tests/run_pyspark_from_build.sh

```diff
@@ -168,8 +168,8 @@ else
 
     if [[ ${TEST_PARALLEL} -lt 2 ]];
     then
-        # With xdist 0 and 1 are the same parallelsm but
-        # 0 is more effecient
+        # With xdist 0 and 1 are the same parallelism but
+        # 0 is more efficient
```

Comment:
nit: after we no longer start Spark on xdist master process the major difference is no longer in efficiency but how the job is submitted

Reply:
Got it, thx~Will rephrase this along some other changes